docs/overview.asciidoc

```diff
@@ -0,0 +1,18 @@
+[id="{p}-overview"]
+== Overview
+
+Built on the Kubernetes Operator pattern, Elastic Cloud on Kubernetes (ECK) extends the basic Kubernetes orchestration capabilities to support the setup and management of Elasticsearch and Kibana on Kubernetes.
+
+With Elastic Cloud on Kubernetes you can streamline all those critical operations, such as:
+
+. Managing and monitoring multiple clusters
+. Scaling cluster capacity up and down
+. Changing cluster configuration
+. Scheduling backups
```

Comment:
We don't really do anything for that

Reply:
They are mentioned in the ECK post, so we can probably keep them in the overview?

Reply:
So it's wrong in two places?

Reply:
I guess this is the current ECK potential. Because we are channeling this info through the post Anurag wrote, I thought we could have reused it in the overview as well. If this is not the case, those lines can simply be removed. 
@anurag, any thoughts on that?

Reply:
Yes +1 on removing

Reply:
Done.